homeassistant/components/volvooncall/binary_sensor.py

```diff
@@ -35,7 +38,9 @@ def __init__(
         """Initialize the sensor."""
         super().__init__(vin, component, attribute, slug_attr, coordinator)
 
-        if self.instrument.device_class in DEVICE_CLASSES:
+        if self.instrument.device_class in {
+            item.value for item in BinarySensorDeviceClass
+        }:
```

Comment:
Just use the validator instead. Pass the data through the validator and it will return the correct result.